pkg/lib/DynamicListForm.jsx

```diff
@@ -42,7 +47,7 @@ export class DynamicListForm extends React.Component {
     }
 
     render () {
-        const { id, label, actionLabel, formclass, emptyStateString, helperText } = this.props;
+        const { id, label, actionLabel, formclass, emptyStateString, helperText, validationFailed, onValidationChange } = this.props;
```

Comment:
Please add these to `propTypes`, in particular whether they are required or optional (see above)